biojava-structure/src/main/java/org/biojava/nbio/structure/EntityInfo.java

```diff
@@ -988,4 +989,18 @@ public void addChain(Chain chain){
 	public void setChains(List<Chain> chains){
 		this.chains = chains;
 	}
+
+	/**
+	 * @return the type
```

Comment:
Would be good to have javadocs for this one. This is an essential part of the refactoring and of the Entity concept.

Also should we have the return type as an Enum `EntityType` instead of a String?